doc/concepts/decor.rst

```diff
@@ -166,7 +167,24 @@ Take for example the built-in mechanism for passive leaky dynamics:
 
 .. _cablecell-ions:
 
-4. Ion species
+.. _cablecell-scaled-mechs:
+
+4. Scaled mechanisms
+~~~~~~~~~~~~~~~~~~~~~
+:ref:`Inhomogenous expressions  <labels-iexpr>` can be applied to mechanisms, to scale parameters on a cable cell.
```

Comment:
Also, if you want to keep `iexpr`, motivate why they are called `inhomogeneous`.